doc/src/manual/getting-started.md

```diff
@@ -109,6 +109,7 @@ julia [switches] -- [programfile] [args...]
 |`-e`, `--eval <expr>`                  |Evaluate `<expr>`|
 |`-E`, `--print <expr>`                 |Evaluate `<expr>` and display the result|
 |`-L`, `--load <file>`                  |Load `<file>` immediately on all processors|
+|`-t`, `--threads {N\|auto`}            |Enable N threads; `auto` sets N to the number of local CPU threads|
```

Comment:
> `auto` sets N to the number of local CPU threads

Can we be a bit vague about this?  Or reserve the word `auto`?  It'd be nice to if `--threads=auto` uses the number of physical cores and also aware of cgroups.

Reply:
Or at least say that the behavior of `auto` might change in the future.

Reply:
That sounds good to me too.

Reply:
Sure. Just FYI, I didn't "invent" `auto` for this, just reused the same implementation as we have for `--procs`.

Reply:
Ah, I missed that. Can the documentation of `--proc` also include that the interpretation of `auto` may change in the future? (i.e., can relaxing `--proc=auto`'s guarantee be considered minor change?)

Though maybe the safest option is to invent a new name other than `auto` when we have the facility to detect the available number of physical cores.